src/java/org/apache/cassandra/service/accord/AccordCommandStores.java

```diff
@@ -43,7 +44,7 @@ public AccordCommandStores(int numShards, Node.Id node, Function<Timestamp, Time
             int index = i;
             executors[i] = Executors.newSingleThreadExecutor(r -> {
                 Thread thread = new Thread(r);
-                thread.setName(CommandStore.class.getSimpleName() + '[' + node + ':' + index + ']');
+                thread.setName(NamedThreadFactory.globalPrefix() + CommandStore.class.getSimpleName() + '[' + node + ':' + index + ']');
```

Comment:
jvm-dtest thing... logs are easier to read when the thread is prefixed with the right info; this is a `""` outside jvm-dtest